src/main/java/com/revature/controllers/ContentController.java

```diff
@@ -92,6 +92,13 @@
 	public ResponseEntity<Set<Link>> getLinksByContentId(@PathVariable int id) {
 		return ResponseEntity.ok(contentService.getLinksByContentId(id));
 	}
+	
+	// return list of  links attached to a given curriculum
+	@GetMapping("/{id}/curriculum")
```

Comment:
the correct mapping for this resource would actually be curriculum/{id}/links

Reply:
i was trying to be consistent with the previous method used to get links but i will change it
- i  will make test for that method and solve conflict and then push it again